### PR TITLE
Removed Validator > Bank POST/PATCH endpoints

### PR DESCRIPTION
**Pull request by tomijaga**
#80 #85 
I removed only the post/patch endpoints

>  BankBlockServices
  ~BankConfirmationServices~
  ~ConnectedBankService~
  ConnectionRequestServices
  UpgradeRequestService

the ones crossed out are the get endpoints

### CODE REVIEW

**Review by Mirch — CHANGES_REQUESTED** [review]
Almost done, but they also need to be removed from the Bank node :) there are some POST endpoints in there as well.

**Comment by tomijaga** [comment]
Removed all Bank > Validator  Post/Patch endpoints

>BankConfirmationBlockService
ConnectionRequestServices
InvalidBlockService
UpgradeNoticeService
ValidatorConfirmationServices

File: src/Thenewboston.Tests/Bank/Api/InvalidBlockServiceTests.cs [hunk 1]
```diff
@@ -1,119 +0,0 @@
...
-                Assert.Equal(3, invalidBankBlocks.Count);
-                Assert.Equal("2bcd53c5-19f9-4226-ab04-3dfb17c3a1fe", invalidBankBlocks.Results.ElementAt(0).Id);
-            }
-            
-            [Fact]
-            public async void GetInvalidBlockWithPageOnlyPaginationReturnedSuccessCodeandData()
-            {
-                var service = BuildBankInvalidServiceMock();
-
-                var invalidBankBlocks = await service.GetInvalidBankBlocksAsync();
```
Comment (Mirch) [inline src/Thenewboston.Tests/Bank/Api/InvalidBlockServiceTests.cs:38]: This is a test for the GET endpoint. We need it.

File: src/Thenewboston/Bank/Api/IInvalidBlocksService.cs [hunk 2]
```diff
@@ -1,14 +0,0 @@
-﻿using System.Threading.Tasks;
-using Thenewboston.Bank.Api.Models;
-using Thenewboston.Bank.Models;
-using Thenewboston.Common.Api.Models;
-
-namespace Thenewboston.Bank.Api
-{
-    public interface IInvalidBlocksService
-    {
-        Task<PaginatedResponseModel<BankInvalidBlock>> GetInvalidBankBlocksAsync(int offset, int limit);
```
Comment (Mirch) [inline src/Thenewboston/Bank/Api/IInvalidBlocksService.cs:10]: We need the GET endpoint.

File: src/Thenewboston/Bank/Api/InvalidBlocksService.cs [hunk 3]
```diff
@@ -1,46 +0,0 @@
...
-    {
-        private readonly IHttpRequestSender _requestSender;
-
-        public InvalidBlocksService(IHttpRequestSender requestSender)
-        {
-            _requestSender = requestSender;
-        }
-
-        public async Task<PaginatedResponseModel<BankInvalidBlock>> GetInvalidBankBlocksAsync(int offset = 0, int limit = 10)
-        {
```
Comment (Mirch) [inline src/Thenewboston/Bank/Api/InvalidBlocksService.cs:23]: Needed.

**Review by Mirch — CHANGES_REQUESTED** [review]

**Review by Mirch — APPROVED** [review]

**Pull request merged**